_zh-cn/tour/regular-expression-patterns.md

```diff
@@ -13,3 +13,50 @@ language: zh-cn
 next-page: extractor-objects
 previous-page: singleton-objects
 ---
+
+正则表达式是一类特殊的字符串，可以用来在数据中寻找特定模式.  任何字符串都可以通过调用 `.r` 方法来转换成正则表达式
```

Comment:
Chinese punctuation "。" should be used instead of "."

Reply:
Yes, thanks a lot.